src/localisation/string_ids.h

```diff
@@ -960,6 +960,7 @@ enum {
 	STR_SELECT_DESIGN = 2307,
 	STR_TRACK_DESIGNS = 2308,
 	STR_BUILD_CUSTOM_DESIGN = 2310,
+	STR_SELECT_OTHER_RIDE = 5453,
```

Comment:
Can you move this so that its in order of string id number. Also can you replace the tab with spaces in the english_uk.txt entry.